Панчо_Паоло/task02/qsort.kt

```diff
@@ -0,0 +1,26 @@
+package ru.bmstu.iu4.hellow
+
+object qsort {
+    @JvmStatic
+    fun main(args: Array<String>) {
+        var temp: Int = 0
+        var intArray = intArrayOf(5, 7, 1, 3, 2, 9, 4)
+
+        var n: Int = intArray.size
+        for (i in 0..(n-2)){
+            for (j in 0..(n-2)){
```

Comment:
No need to store array size in separated value, just
`for (i in 0 until (intArray.size - 1)) ....`